src/components/Books/Books.jsx

```diff
@@ -1,16 +1,23 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
+import { v4 as uuidv4 } from 'uuid';
 import PropTypes from 'prop-types'
 import Book from './Book';
 import NewBook from './NewBook';
-import { addBook, removeBook } from '../../redux/books/booksSlice';
+// import { addBook, removeBook } from '../../redux/books/booksSlice';
```

Comment:
- [ ] Please do not leave unused code as comments in your repo. They make the code base dirty and can lead to errors which are hard to debug 😺